packages/backend/src/app.ts

```diff
@@ -91,6 +91,28 @@ export type ListContext = Context<
   AppRequest<'accountId', never, PageQueryParams>
 >
 
+export interface DatabaseCleanupRule {
```

Comment:
I think you added this to the wrong package. This is `backend` and not `auth`.

Reply:
☝️